recipes/classy_vision/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "classy_vision" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # NOTE: the PyPI sdist is broken for 0.1.0, use PyPI for future versions
+  url: https://github.com/facebookresearch/ClassyVision/archive/{{ version }}.tar.gz
+  sha256: e6cbb9a68a3e8ece4365f006dcb0823df85ec8b860270af5cd752f2f7ea6cbe2
+
+build:
+  number: 0
+  skip: True  # [py<36 or not linux64]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
```

If (and only if!) it is made noarch:
```suggestion
    - python >=3.6
```